sdk/highlightinc-highlight-datasource/provisioning/datasources/datasource.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: 1
+
+datasources:
+  - name: Highlight.io
+    type: highlightinc-highlight-datasource
+    jsonData:
+      clientID: ''
+      projectID: 1344
+      tokenURL: 'https://pri.highlight.io/oauth/token'
+      backendURL: 'https://pri.highlight.io'
```

Comment:
does our api just work for the demo project, or do we need to add some special logic to support it?

Reply:
it just works! if no oauth credentials are entered into the data source, i'm using the default http client rather than the oauth client, and the auth in the resolvers lets you query for demo project data